public/js/specberus.js

```diff
@@ -54,13 +54,33 @@ jQuery.extend({
     ,   done = 0
     ,   result = {exceptions: [], errors: [], warnings: [], infos: []}
     ,   total = 0
+    ,   running = false
     ;
 
     // handshake
     socket.on("handshake", function (data) {
         console.log(`Handshake; using version “${data.version}”.`);
+
+        socket.on('disconnect', () => {
+            socket.close();
+            toggleForm(false);
+            window.location.href = window.location.href;
+        });
+
     });
 
+    const toggleForm = (bool) => {
+        if (bool) {
+            $('form').css('opacity', 1);
+            $('form input').removeClass('disabled').removeAttr('disabled');
+            $('button[type=submit]').text('Check');
+        } else {
+            $('form').css('opacity', 0.25);
+            $('form input').addClass('disabled').attr('disabled', 'disabled');
```

Comment:
While specberus is running, it is still possible to update the form. `<label>`s and `<select>` should be disabled as well.